### DISCUSSION TITLE
关于Tabs的命名，吐槽一下

### DISCUSSION THREAD

**Discussion by cx5168** [ant-design/ant-design#33685] [Ideas]
组件Tabs的子组件是TabPane，不是应该命名为TabPanel更合理吗？renderTabBar的时候，习惯性的写成props.panels去拿数据死活undefined，后来才发现应该是props.panes。
吐槽一下！！！

**Comment by afc163** [comment]
这个命名已经不可考，前 React 时代 Pane 和 Panel 的命名都有，antd 这个可能是从 https://plugins.jquery.com/pane/ 来的。
